mail_tracking/models/mail_thread.py

```diff
@@ -40,3 +51,48 @@ def message_get_suggested_recipients(self):
                         record._message_add_suggested_recipient(
                             res, partner=partner, reason=_('Cc'))
         return res
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        """Add a filter to any model with mail.thread that will show up records
```

Comment:
... and now that we speak about docstrings, in the case of [multiline docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings):

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

An idea of what would be a proper multiline docsting here:

```python
"""Add filters for failed messages.

These filters will show up on any form or search views of any
model inheriting from ``mail.thread``.
"""
```

I guess you didn't know there's a PEP on docstrings and you're gonna like to have a standard on that. :blush: